dockstore-webservice/pom.xml

```diff
@@ -938,7 +938,13 @@
         <dependency>
             <groupId>org.glassfish.jaxb</groupId>
             <artifactId>jaxb-runtime</artifactId>
-            <version>2.3.3</version>
+            <version>2.3.1</version>
+            <scope>runtime</scope>
```

Comment:
Any special reason to move to an older version of jaxb-runtime (2.3.3->2.3.1)?

Reply:
> Any special reason to move to an older version of jaxb-runtime (2.3.3->2.3.1)?

The enforcer plugin was causing a failure after I upgraded the Hibernate version.